### ISSUE TITLE
WCAG 4.1.2: Ensure buttons have discernible text (#script-selector,.combobox-button)

### ISSUE THREAD

**Issue by pelikhan** [vscode-elements/elements#320]
Title: Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensure buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

Target application: GenAIScript Script Runner - http://127.0.0.1:8003/?scriptid=add.test

Element path: #script-selector,.combobox-button

Snippet: <button class="combobox-button" type="button">

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Microsoft Edge version 133.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by pelikhan** [comment]
![Image](https://github.com/user-attachments/assets/5da44220-683a-4ca7-80ad-52c6397d41df)
